opencga-app/app/scripts/azure/arm/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```

Comment:
Why have you gitignored the lock file? This should normally be committed, right?

Reply:
yeah this deserves some explanation in the file. To make 'validate' work I need to install 'armval', so I don't mess with peoples machines I install the tool in this directory which means `node_modules` and `package-lock` get created... however as this has nothing to do with our code I've and only affects validation I've ignored them

Reply:
removed as now using npx :)